### PR TITLE
LPD-35460 Import data and pages in Classic set initializer

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-35460

### CODE REVIEW

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json
Comment (dsitu) [inline modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json:1]: This is just a straight copy over from minium. We will need to update the SKU naming and images.

Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json:1]: No worries for those @dsitu, we can temporarily reuse them until we have a replacement.

**Comment by dsitu** [comment]
failed test are unrelated to changes

**Comment by gianmarcobrunialti** [comment]
Hey @dsitu I've test your PR and it looks good, since we're going for B2B first, could you also set page permissions to have all pages enclosed behind login? Thanks!

**Comment by dsitu** [comment]
@gianmarcobrunialti pages are not restricted to site members only https://github.com/liferay-commerce/liferay-portal/pull/5436/commits/0334ff57235fc92f25ebcbcae45a2461818f666f

**Review by gianmarcobrunialti — APPROVED** [review]

**Comment by gianmarcobrunialti** [comment]
Yay 🎉

**Comment by dsitu** [comment]
ci:forward

**Comment by dsitu** [comment]
ci:forward:force

**Pull request closed without merging**